Adyen/UI/Styles/ApplePayStyle.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+import PassKit
+import UIKit
+
+///  Contains the styling options for Apple Pay components.
+public struct ApplePayStyle {
+    
+    /// Style of the payment button. When nil, it's set to automatic based on Dark or Light Mode.
+    public var paymentButtonStyle: PKPaymentButtonStyle?
+    
+    /// Type of the Apple Pay payment button. Default is `.plain`
+    public var paymentButtonType: PKPaymentButtonType
+    
+    /// Corner radius for the payment button. iOS 12 or above. Defaults to 4 points.
+    public var cornerRadius: CGFloat
+    
+    /// Stying for the label that contains the formatted amount text.
+    public var hintLabelStyle: TextStyle
```

Comment:
there is no need for the `Style` postfix, since the enclosing object has the word "style" in its name.